### ISSUE TITLE
Issue with CSS property in built-in DMG Theme and PHB theme GitHub

### ISSUE THREAD

**Issue by Sirnyght** [naturalcrit/homebrewery#3274]
### Renderer

v3

### Browser

Chrome

### Operating System

Windows

### What happened?

Hello,

While styling my document and trying the different features, I noticed that some CSS in the default PHB theme and in the built-in DMG theme prevented custom styling

It appears that the CSS property `all: unset` prevents custom styling by the user. It can be found at lines 542, 636, 682, and 703 of the PHB theme and at line 25 of the DMG theme.

It's not really that much of a problem in the default theme, the case where the property would apply aren't case where you would normally customize styling. However, it's more of a problem in the DMG theme, where it prevents custom styling of the first line of quote blocks.

I don't know if it's intended, but I figured it'd be better to let you know about that even if built-in themes are an alpha feature, given that the bit of code preventing styling is also present in the default PHB theme.

tl;dr: CSS property `all: unset` can't be overridden by user.

The following code snippet doesn't work with the DMG theme, it's the simplest example.

### Code

```gfm
.quote.tome p::first-line {
  font-variant 		: small-caps;
  font-family 		: BookInsanityRemake;
  font-size		: 15px;
  text-transform	: lowercase;
  line-height		: 14px;
}
```

**Comment by 5e-Cleric** [comment]
sure does, it only lacks specificity:
```
.page .quote.tome p::first-line {
  font-variant 		: small-caps;
  font-family 		: BookInsanityRemake;
  font-size		: 15px;
  text-transform	: lowercase;
  line-height		: 14px;
}
```
